### ISSUE TITLE
can i report track multiple metric using nni.report_intermediate_result?

### ISSUE THREAD

**Issue by hzhz2020** [microsoft/nni#4629]
**Describe the issue**:
I would like to track multiple metric during training, can i use nni.report_intermediate_result to track them and have them all display in the WebUI?

**Environment**:
- NNI version: 2.6
- Training service (local|remote|pai|aml|etc):
- Client OS:
- Server OS (for remote mode only):
- Python version: 3.8
- PyTorch/TensorFlow version: 
- Is conda/virtualenv/venv used?:
- Is running in Docker?:

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:

**Log message**:
 - nnimanager.log:
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**How to reproduce it?**:

**Comment by Lijiaoa** [comment]
@hzhz2020 https://github.com/microsoft/nni/discussions/4289 may help you.
